regex-assembly/920220-chain1.ra

```diff
@@ -0,0 +1,15 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+
+##!^ ^
+##!$ $
+
+##! grab the path, except for the lat path segment (separate rule)
+(.*)/
+##!=>
+##! skip the last path segment, if there is one
+(?:[^?]+)?
```

Comment:
I think it worthwhile to explicitly state this is a non-capturing group since you are going to refer to the captured groups in the chained rules afterwards.